maquetado/scss/footer.scss

```diff
@@ -0,0 +1,28 @@
+@import 'styles';
+@import 'colors';
+.footer {
+  align-items: center;
+  background-color: $pictonBlue;
+  display: flex;
+  flex-direction: column;
+  padding: 40px 20px 20px;
+  z-index: 1;
```

Comment:
you can define the color here if all the children have white text (it is inherited by default)